src/test/java/madalinapopescu/tests/tema8/teste/RefreshPageTest.java

```diff
@@ -0,0 +1,18 @@
+package madalinapopescu.tests.tema8.teste;
+
+import madalinapopescu.tests.tema8.pageobjects.RefreshPage;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class RefreshPageTest extends BaseTest {
+    @Test(groups = {"mobile", "desktop"})
+    public void refreshTest() {
+        driver.get("https://testpages.herokuapp.com/styled/refresh");
+        RefreshPage refreshPage = new RefreshPage(driver);
+        driver.navigate().refresh();
+        Assert.assertEquals(refreshPage.getStatusForTimestamp(), refreshPage.getStatusForTimestamp(), "Timestamp has been changed!");
+    }
+}
+
```

Comment:
One blank line is enough. You can remove the extra ones.

Reply:
Iar spatii.....